examples/tutorials/README.md

```diff
@@ -27,28 +30,46 @@ practitioners. By enabling the growth of open source tools for drug discovery,
-* [Part 19: Large Scale Chemical Screens](19_Large_Scale_Chemical_Screens.ipynb)
-* [Part 20: ConvertingDeepChem Models to TensorFlow Estimators](20_Converting_DeepChem_Models_to_TensorFlow_Estimators.ipynb)
-* [Part 21: Introduction to Bioinformatics](21_Introduction_to_Bioinformatics.ipynb)
-* [Part 22: Using HuggingFace + Transfer Learning for Toxicity Predictions](22_Transfer_Learning_With_HuggingFace_tox21.ipynb)
+The tutorial is organized as follows:
 
+### Introduction to DeepChem
+The following tutorials covers the core features of DeepChem. Covering these
+tutorials will help you in getting started with DeepChem for machine learning. The later
+tutorials discuss about using DeepChem for specific applications.
 
+* [1.1 The Basic Tools of the Deep Life Sciences](01_The_Basic_Tools_of_the_Deep_Life_Sciences.ipynb)
+* [1.2 Working With Datasets](02_Working_With_Datasets.ipynb)
+* [1.3 An Introduction to MoleculeNet](03_An_Introduction_To_MoleculeNet.ipynb)
+* [1.4 Working With Splitters](08_Working_With_Splitters.ipynb)
+* [1.5 Creating Models with Tensorflow and Pytorch](05_Creating_Models_with_TensorFlow_and_PyTorch.ipynb)
+* [1.6 Advanced Model Training](09_Advanced_Model_Training.ipynb)
+* [1.7 Putting Multitask Learning to Work](11_Putting_Multitask_Learning_to_Work.ipynb)
+* [1.8 Creating a high fidelity model from experimental data](10_Creating_a_high_fidelity_model_from_experimental_data.ipynb)
+* [1.9 Using Reinforcement Learning to Play Pong](26_Using_Reinforcement_Learning_to_Play_Pong.ipynb)
+* [1.10 Training a Generative Adversarial Network on MNIST](15_Training_a_Generative_Adversarial_Network_on_MNIST.ipynb)
+* [1.11 Conditional Generative Adversarial Networks](14_Conditional_Generative_Adversarial_Networks.ipynb)
+* [1.12 Introduction to Graph Convolutions](06_Introduction_to_Graph_Convolutions.ipynb)
+* [1.13 Uncertainty in Deep Learning](25_Uncertainty_In_Deep_Learning.ipynb)
+* [1.14 Introduction to Model Interpretability](23_Introduction_to_Model_Interpretability.ipynb)
+* [1.15 Large Scale Chemical Screens](19_Large_Scale_Chemical_Screens.ipynb)
+
+### Molecular Machine Learning
+* [2.1 Molecular Fingerprints](04_Molecular_Fingerprints.ipynb)
+* [2.2 Going Deeper on Molecular Featurizations](07_Going_Deeper_on_Molecular_Featurizations.ipynb)
+* [2.3 Learning Unsupervised Embeddings for Molecules](16_Learning_Unsupervised_Embeddings_for_Molecules.ipynb)
+* [2.4 Synthetic Feasibility Scoring](24_Synthetic_Feasibility_Scoring.ipynb)
+* [2.5 Modeling Protein Ligand Interactions](13_Modeling_Protein_Ligand_Interactions.ipynb)
+* [2.6 Modeling Protein Ligand Interactions with Atomic Convolutions](14_Modeling_Protein_Ligand_Interactions_With_Atomic_Convolutions.ipynb)
+* [2.7 Atomic Contributions for Molecules](28_Atomic_Contributions_for_Molecules.ipynb)
+* [2.8 Transfer Learning with ChemBERTa Transformers](22_Transfer_Learning_With_ChemBERTa_Transformers.ipynb)
+* [2.9 Protein Deep Learning](30_Protein_Deep_Learning.ipynb)
```

Comment:
Let's move the protein tutorials into their own section since proteins are quite different from small molecules